torq.sh

```diff
@@ -224,6 +224,13 @@ usage() {
  }
 
 
+if [ -f $SETENV ]; then                                                                             # check script exists 
+  . ./$SETENV                                                                                       # load the environment
```

Comment:
this still requires `$SETENV` to be located in current working directory, I think it'd be more generic if the default value of `SETENV` was something like `$(pwd)/setenv.sh` and this line was just `. $SETENV` to allow people to use a different location if they want - they'd maybe have to have a more custom `setenv.sh` for that to still work, but we shouldn't exclude the option.

Reply:
This works fine when I move `setenv.sh` out of the current working directory and provide the path from the working directory to the file. However, your suggestion is a better idea.